csharp-basic-topics/ValueAndReferenceTypes/ValueAndReferenceTypes/ValueTypeSample.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace ValueAndReferenceTypes
+{
+    public struct ValueTypeRectangle
+    {
+        public int breadth;
+        public int length;
+        public int area;
```

Comment:
These should be public properties instead of public fields like this : 

```C#
        public int Breadth { get; set;}
        public int Length { get;  set;}
```